### PR TITLE
DDF-1996 Add integration tests for testing spatial, temporal and contextual CSW queries

### PR DESCRIPTION
**Pull request by roelens8**
#### What does this PR do?
Adds integration tests for testing spatial, temporal and contextual CSW queries.
#### Who is reviewing it?
@ryeats @tbatie @jckilmer @bcwaters @coyotesqrl 
#### How should this be tested?
Run TestSpatial and verify all tests pass
#### Any background context you want to provide?
N/A
#### What are the relevant tickets?
N/A
#### Screenshots (if appropriate)
N/A
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [X] Update / Add Integration Tests

### CODE REVIEW

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java [hunk 1]
```diff
@@ -564,6 +568,32 @@ protected Option installStartupFile(InputStream resourceInputStream, String dest
         return replaceConfigurationFile(destination, tempFile);
     }
 
+
+    /**
+     * Turns off all logging.
+     *
+     * @throws IOException
+     */
+    public static void turnOffLogging(ConfigurationAdmin configAdmin) throws IOException {
```
Comment (bcwaters) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java]: configAdmin is already available within AbstractIntegrationTest so I don't think this argument is not needed

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCsw.java [hunk 2]
```diff
@@ -0,0 +1,478 @@
...
+        return DocumentBuilderFactory.newInstance()
+                .newDocumentBuilder()
+                .parse(new ByteArrayInputStream(input.getBytes()));
+    }
+
+    public enum ResultType {
+        TITLE, COUNT
+    }
+
+    public static class ExpectedResultPair {
```
Comment (bcwaters) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCsw.java]: Pretty sure this class no longer needs to be static, remnant of an old approach for the test.

**Comment by bcwaters** [comment]
:thumbsup:

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java (hunk 1 above)
Comment (roelens8) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java]: Fixed.

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCsw.java (hunk 2 above)
Comment (roelens8) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCsw.java]: Fixed.

**Comment by ryeats** [comment]
:+1:

**Comment by tbatie** [comment]
:+1:

**Comment by coyotesqrl** [comment]
:+1:

**Comment by tbatie** [comment]
Hero'd again just to be safe, all tests passed.

**Pull request merged**